core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -503,8 +503,9 @@ object Invoker {
         })
 
         val port = config.servicePort.toInt
+
         BasicHttpService.startService(actorSystem, "invoker", "0.0.0.0", port, new Creator[InvokerServer] {
-            def create = new InvokerServer(invokerInstance, config.invokerInstances.toInt)
+            def create = new InvokerServer(invokerInstance, config.invokerInstances.toInt, port)
```

Comment:
port passed twice (i know to different constructors) - would be nice to avoid if doable.